docs/DeveloperGuide.md

```diff
@@ -309,28 +309,67 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Add iteration to a commission**<br>
+**Guarantees:**
+* An iteration, with an associated image, will be added to the specified commission only if its creation is successful
+* The new iteration will not be created if the image provided is not valid (e.g. it does not exist, or it is not
+  of the correct image format)
 
-**MSS**
+**MSS:**
+1. User <u>opens a commission they want to edit (Open Commission)</u>.
+2. User enters the command to add an iteration to the commission selected, specifying the image to use for the
+   iteration using an absolute file path.
+3. AB creates a new iteration for the commission, associated with the image specified.
+4. AB updates the GUI to also show the new iteration.
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+   Use case ends.
 
-    Use case ends.
+**Extensions:**
+* 2a. The user enters a command that AB does not recognise.
+    * 2a1. AB shows an error message, and asks the user to enter a correct command.
+    * 2a2. User enters a new command.
 
-**Extensions**
+      Steps 2a1 and 2a2 are repeated until the command is correctly entered.
 
-* 2a. The list is empty.
+      Use case resumes from step 3.
 
-  Use case ends.
+* 2b. The specified file path the user specifies is invalid or does not exist.
+    * 2b1. AB shows an error message, and asks the user for a new command with a valid file path that exists.
+    * 2b2. User enters a new command.
 
-* 3a. The given index is invalid.
+      Steps 2b1 and 2b2 are repeated until the command is correctly entered.
 
-    * 3a1. AddressBook shows an error message.
+      Use case resumes from step 3.
 
-      Use case resumes at step 2.
+* 2c. The specified file under the file path is not a valid file type.
+    * 2c1. AB lets the user know that the file is not supported, and requests for a new command
+      with a file of support file types.
+    * 2c2. User enters a new command.
+
+      Steps 2c1 and 2c2 are repeated until the command is correctly entered.
+
+      Use case resumes from step 3.
+
+Use case: Delete an iteration of a commission<br>
```

Comment:
```suggestion
**Use case: Delete an iteration of a commission<br>
```

Missing starting `**` is causing it to not render correctly.

Reply:
Woops thanks!